app/Repositories/HistoryRepository.php

```diff
@@ -11,6 +11,7 @@ public function __construct()
 
     protected function getEntityFields(): array
     {
-        return ['date', 'first_operand', 'second_operand', 'sign', 'result'];
+        return ['user', 'date', 'first_operand', 'second_operand', 'sign', 'result'];
```

Comment:
let's save username here

```suggestion
        return ['user_name', 'date', 'first_operand', 'second_operand', 'sign', 'result'];
```